node/pkg/status/nodestatus_test.go

```diff
@@ -48,6 +51,9 @@ var _ = Describe("Node status FV tests", func() {
 	Expect(err).NotTo(HaveOccurred())
 	cfg.Spec = apiconfig.CalicoAPIConfigSpec{
 		DatastoreType: apiconfig.Kubernetes,
+		KubeConfig: apiconfig.KubeConfig{
+			Kubeconfig: os.Getenv("KUBECONFIG"),
+		},
```

Comment:
I am not sure this change is really needed, but on my local environment, without this change `make fv` would silently skip whole nodestatus test suite, giving a false-positive success, while actually 0 tests were executed.